reference/api/search.md

```diff
@@ -41,6 +41,14 @@ This is the preferred route to perform search when an API key is required, as it
 
 [Learn more about how to use the search parameters](/reference/features/search_parameters.md).
 
+#### Placeholder search
+
+When the `q` parameter is not specified, a [placeholder](/reference/features/search_parameters.md#query-q) search is run instead.
```

Comment:
```suggestion
When no search query is specified, a [placeholder search](/reference/features/search_parameters.md#query-q) is run instead.
```

Reply:
Not relevant to this PR, but I want to make this change since `q` parameter isn't really exposed to users of the SDKs (they have dedicated search functions).